.github/workflows/deploy.yml

```diff
@@ -27,6 +27,16 @@ jobs:
           cache: yarn
           cache-dependency-path: website/yarn.lock
 
+      - name: Cache code outputs
+        id: cache-code-outputs
+        uses: actions/cache@v3
+        env:
+          cache-name: cache-code-outputs
+        with:
+          path: website/build/solutions
+          key: 1 # currently using a constant as a key
```

Comment:
use a name that is not a number; this may trigger some arcane bugs if they use it for file names